app/group/serializers/fine.py

```diff
@@ -114,3 +115,28 @@ class Meta:
 
     def get_user(self, obj):
         return DefaultUserSerializer(obj).data
+
+
+class FineStatisticsSerializer(BaseModelSerializer):
+
+    payed = serializers.SerializerMethodField()
+    approved_and_not_payed = serializers.SerializerMethodField()
+    not_approved = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Group
+
+        fields = ("payed", "approved_and_not_payed", "not_approved")
+
+    def get_sum(self, obj, kwargs):
+        sum = obj.fines.filter(**kwargs).aggregate(sum=Sum("amount"))["sum"]
+        return sum if sum else 0
+
+    def get_payed(self, obj):
+        return self.get_sum(obj, {"payed": True})
```

Comment:
Funker ikke dette også? Mer clean syns jeg kanskje isåfall

```suggestion
    def get_sum(self, obj, *args, **kwargs):
        sum = obj.fines.filter(*args, **kwargs).aggregate(sum=Sum("amount"))["sum"]
        return sum if sum else 0

    def get_payed(self, obj):
        return self.get_sum(obj, payed=True)
```

Reply:
alt er lov i python så klart det gikk